sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/faultinjection/FaultInjectionCondition.java

```diff
@@ -1,13 +1,16 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.cosmos.models;
+package com.azure.cosmos.faultinjection;
+
+import com.azure.cosmos.util.Beta;
 
 /***
  * Fault injection condition.
  * A fault injection rule will not be applicable if the condition mismatches.
  */
-public class FaultInjectionCondition {
+@Beta(value = Beta.SinceVersion.V4_42_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
```

Comment:
Hmm - I am not too keen to keep Beta attribute on the FaultInjection model classes - I don't think we ever need to break them - so, would have kept Beta off of them. Only the actual injection my confidence level is lower - so,  would keep Beta tag - but I guess you think all or nothing :-). Let's chat about it when you are online. I would rather not use Beta-tag than put it everywhere.

Reply:
Discussed offline, only keep Beta in the CosmosFaultInjectionHelper